docs/DeveloperGuide.md

```diff
@@ -1057,12 +1057,15 @@ Use case ends.
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 patients without a noticeable sluggishness in performance for typical usage.
+2.  Should be able to hold up to 1000 patients and 1000 appointments without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4. Should work without internet access i.e offline-only.
+5. Should not take more than 25MB of space, including data files during normal usage.
+6. Should consistently save and load data locally so that import data in different devices should return the same application state.
```

Comment:
Sorry I dont understand point 6, initially you mentioned that the data should be offline-only but the data in different devices should return the same state?

Reply:
> Sorry I dont understand point 6, initially you mentioned that the data should be offline-only but the data in different devices should return the same state?

It is from different devices
omg

Reply:
why you omg haha